packages/core/admin/admin/src/pages/SettingsPage/components/Tokens/DeleteTokenDialog/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ConfirmDialog } from '@strapi/helper-plugin';
+
+export const DeleteTokenDialog = ({ onClose, onConfirm, isOpen }) => {
+  return <ConfirmDialog onToggleDialog={onClose} onConfirm={onConfirm} isOpen={isOpen} />;
+};
```

Comment:
Do you really need this component now that it is basically `ConfirmDialog`?

Reply:
replaced with ConfirmDialog